pilgrimage.lic

```diff
@@ -34,6 +34,7 @@ class Pilgrimage
       bput("wear my #{description}", 'You put on')
       move('out')
     else
+      continue if DRStats.paladin?
```

Comment:
What does this line do?

Reply:
Skips the error message and exit so we can do the rest of the script. I did it this way because then we'd need to pull the failure logic out and put it in the main, this just follows suit.

Reply:
The only way to get into this block is if you don't have a badge. Why do you want to go visit all the shrines if you are a Paladin without a badge?

Reply:
Yeah, I didn't thoroughly read this script enough.